src/config/logger.ts

```diff
@@ -11,17 +11,17 @@ const outputMode = process.env.MICROS_ENV ? "json" : "short";
 // add levelInString to include DEBUG | ERROR | INFO | WARN
 const formatOut = filteringStream(bformat({ outputMode, levelInString: true }));
 
-const requestSerializer = (req: Request) => (!req || !req.connection) ? req : {
+const requestSerializer = (req: Request) => req && {
 	method: req.method,
 	url: req.originalUrl || req.url,
 	path: req.path,
 	headers: req.headers,
-	remoteAddress: req.connection.remoteAddress,
-	remotePort: req.connection.remotePort,
+	remoteAddress: req.connection?.remoteAddress,
+	remotePort: req.connection?.remotePort,
 	body: req.body
 };
 
-const errorSerializer = (err) => (!err || !err.stack) ? err : {
+const errorSerializer = (err) => err && {
```

Comment:
What does this && operator doing? 

If err is not set then it will be undefined, otherwise it will be the following object?

Reply:
if error is truthy it continues to return the object or else it returns itself which will be serialized out.